.github/actions/update-releases-md/action.yml

```diff
@@ -0,0 +1,13 @@
+name: 'Update releases documentation'
+description: 'Update releases documentation'
+inputs:
+  releases_json_file:
+    description: 'The input releases json file.'
```

Comment:
nit: To be consistent with `releases_md_file`, should this be 'The input releases json file **path**'?

Reply:
I believe this would also apply for update-releases-json